datashuttle/utils/folders.py

```diff
@@ -259,12 +260,15 @@ def search_sub_or_ses_level(
     sub : either a subject name (string) or None. If None, the search
         is performed at the top_level_folder level
 
-    ses: either a session name (string) or None, This must not
+    ses : either a session name (string) or None, This must not
         be a session name if sub is None. If provided (with sub)
         then the session folder is searched
 
-    str: glob-format search string to search at the
+    str : glob-format search string to search at the
         folder level.
+
+    verbose : If `True`, if a search folder cannot be found, a message
```

Comment:
I was confused by the wording, is this better?
```suggestion
    verbose : If `True`, when a search folder cannot be found, a message
          will be printed with the missing path.
```